diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/KdocMethodsTest.kt

```diff
@@ -339,4 +339,17 @@ class KdocMethodsTest : LintTestBase(::KdocMethods) {
                 LintError(2, 3, ruleId, "${KDOC_TRIVIAL_KDOC_ON_FUNCTION.warnText()} Returns X", false)
         )
     }
+
+
+    @Test
+    @Tag(WarningNames.KDOC_TRIVIAL_KDOC_ON_FUNCTION)
+    fun `should not trigger on override funcs`() {
+        lintMethod(
+                """
+                    |class Some : A {
+                    |   override fun foo() {}
```

Comment:
```suggestion
                    |   override fun foo() {}
                    |
                    |   override fun bar(t: T): U { return U() }
```
Because we have separate warnings when params and return type are not documented, so we should have another example to ensure they are not triggered.